config/environments/development.rb

```diff
@@ -31,14 +31,17 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise an error on page load if there are pending migrations.
+  # config.active_record.migration_error = :page_load
+
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = false
 
   # Asset digests allow you to set far-future HTTP expiration dates on all assets,
   # yet still be able to expire them through the digest params.
-  config.assets.digest = false
+  config.assets.digest = true
```

Comment:
This concerns me. Have you been able to test this at all? We don't use the asset pipeline in development.